src/main/java/frc/robot/config/Config.java

```diff
@@ -68,6 +72,11 @@ private Config() {
     public static final int CAN_TIMEOUT_SHORT = 10;
     public static final int CAN_TIMEOUT_LONG = 100;
 
+    // Fluid constants
+    static NetworkTable constantsTable = NetworkTableInstance.getDefault().getTable("constants");
+    public static FluidConstant<Double> INTAKE_SPEED = new FluidConstant<>("intake-target-speed", 0.25d)
+            .registerToTable(constantsTable);
```

Comment:
Is it possible to define this constant inside the Intake subsystem and still have it be treated as a fluid constant?

I guess what I'm asking is if all constants for all subsystems should be defined in one common and global Config class or whether it makes more sense to define them inside the subsystems that use them.

Defining them in one spot makes them easy to find, but that breaks the encapsulation concept of having things contained in Subsystems.

Reply:
Yes, it's possible to define them wherever you'd like. I originally put them in the config file because that is how it was done last year, although I would prefer having them in the subsystem classes.